ui/src/components/Projects/Create/index.tsx

```diff
@@ -91,33 +93,34 @@ const noTeam = (projectId: string) => {
   );
 };
 
-const addTeam = (projectId: any) => {
-  const teams = [
-    {
-      banner: '',
-      id: '122',
-      issue_link_template: '',
-      logo_square: '',
-      name: 'AB',
-      pr_link_template: '',
-      project_id: '',
-    },
-  ];
+const addTeam = (p: Props) => {
+  const teams = useSelector(teamSelector.selectAll);
+  const listItems: ListItemProps[] = teams.map((t) => ({
```

Comment:
We should rename this variable to `teamListItems` or something better. With just `listItems, this variable being in `Projects/Create` page can't infer its value from context clearly enough.